src/test/java/core/basesyntax/strategy/impl/AddOperationServiceImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AddOperationServiceImplTest {
+    private AddOperationServiceImpl addOperationService;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
```suggestion
    public void setUp() {
```
remove throws Exception from everywhere